menuFetcher.ts

```diff
@@ -47,33 +46,32 @@ export class MenuFetcher {
         }
         this._runningRequests[url] = [doneCallback];
 
-        const options = {
-            url,
-            method: "GET",
-            headers: { // some sites need us to pretend to be a browser to work
+        const done = (e: Error, m: IMenuItem[]) => {
+            const doneCallbacks = this._runningRequests[url];
+            delete this._runningRequests[url];
+            if(e) {
+                console.error("Error for %s: %s", url, e);
+            }
+            doneCallbacks.forEach(dc => dc(e, m));
+        };
+
+        Axios.get<string>(url, {
+            method: "get",
+            headers: {
                 "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)",
-                "Accept": "text/html,*/*",
-                "Accept-Language":"sk" // we want response in slovak (useful for menu portals that use localization, like zomato)
+                Accept: "text/html,*/*",
+                "Accept-Language": "sk", // we want response in slovak (useful for menu portals that use localization, like zomato)

```

Comment:
there should be a rule for trailing comas in eslint... i think we had it with tslint

Reply:
will check

- [ ]

Reply:
Ok, added rule to not require them.
In the past I was also against trailing commas for multilines until I saw the benefit during merging and diff calculations, but I have no troubles to not have them here.